proto/config/edge_node_cluster.proto

```diff
@@ -0,0 +1,37 @@
+// Copyright(c) 2024 Zededa, Inc.
+// All rights reserved.
+
+syntax = "proto3";
+
+import "evecommon/evecommon.proto";
+import "evecommon/acipherinfo.proto";
+
+package org.lfedge.eve.config;
+option go_package = "github.com/lf-edge/eve-api/go/config";
+option java_package = "org.lfedge.eve.config";
+
+message EdgeNodeCluster {
+  // cluster name, in case it has multiple cluster on the same site
+  string cluster_name = 1;
+
+  // cluster UUID
+  string cluster_id = 2;
+
+  // Cluster-Interface, for example "eth1"
+  string cluster_interface = 3;
+
+  // the 'cluster-prefix' IP address of the 'Cluster-Interface', 10.244.244.2/28
+  string cluster_ip_prefix = 4;
+
+  // This device is 'seed-server'
+  bool is_seed_server = 5;
+
+  // This device is an 'Agent' node
+  bool is_agent = 6;
+
+  // SSIP = Seed-ServerIP address to join during bootstrap. E.g. 10.244.244.1
+  string seed_server_ip = 7;
```

Comment:
maybe change this item name to server_joinning_ip,
and add comment saying 'During bootstrap, the device has the cluster_ip_prefix IP address the same as the 'server_joinning_ip', it is responsible for the cluster-init operations.' or something like that.

Reply:
That is an interesting approach. Would that cover every use case? If so, then it is good. I personally like it when we are explicit, like, `cluster_init = true`, but that really is just personal preference, so it doesn't matter.

Reply:
changed field name to join_server_ip